tests/test_runtimes.py

```
@@ -337,7 +333,7 @@ def test_policy_prefers_runtime_override(self):
         engine = RuntimeRoutingPolicyEngine(reg, health, policy=policy)
 
         spec = TaskSpec(task_id="t4", instruction="write code", task_type="code_generation")
-        result, decision = asyncio.run(engine.route_and_execute(spec))
+        result, decision = await engine.route_and_execute(spec)
```

Ruff (0.15.12)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)